xml/obs_supported_formats.xml

```diff
@@ -274,5 +274,11 @@
   <remark>toms 2017-08-18: integrate content from  obs_build_preinstall.xml;
   using a _preinstallimage file</remark>
  </sect1>
+
+ <sect1 condition="tbd">
+  <title>Mkosi Image Builds</title>
+  <para/>
+  <remark>TBD</remark>
+ </sect1>
```

Comment:
I would uncomment it. IMHO, it's not a good idea to have documentation with "TBD" in it as it gives an unprofessional impression and people might be asking if it's still a draft.

```suggestion
<!--
 <sect1 condition="tbd">
  <title>Mkosi Image Builds</title>
  <para/>
  <remark>TBD</remark>
 </sect1>
 -->
```

Reply:
ok, I just copied it as there's many other entires in the same format, but commented it out